source/getting-started/use-cases/container-security.rst

```diff
@@ -25,7 +25,7 @@ Organizations can enhance the security of their containerized applications by mo
 Monitoring the container engine 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Wazuh captures real-time events performed by the Docker engine via its :doc:`Docker listener </user-manual/capabilities/container-security/monitoring-docker>` module. This ensures that no crucial Docker event or operation goes undetected.
+Wazuh captures real-time events performed by the Docker engine via its :doc:`Docker </user-manual/capabilities/container-security/monitoring-docker>` module. This ensures that no crucial Docker event or operation goes undetected.
```

Comment:
**thought**: this references to the module of the Wazuh manager, not to the Docker application of the Wazuh dashboard. Maybe it should not be changed.